graphics/nxwm/src/cnxterm.cxx

```diff
@@ -500,27 +500,18 @@ int CNxTerm::nxterm(int argc, char *argv[])
   // console).  (2) Don't bother trying to put debug instrumentation in the
   // following because it will end up in the NxTerm window.
 
-  std::fflush(stdout);
-  std::fflush(stderr);
-
 #ifdef CONFIG_NXTERM_NXKBDIN
-  std::fclose(stdin);
+  std::fflush(stdin);
```

Comment:
Using fflush with stdin is not appropriate.  fflush has no effect on input streams.  See https://pubs.opengroup.org/onlinepubs/007904975/functions/fflush.html

It would be better to remove this and all other places where fflush is used with stdin.

Reply:
Done.